bcipy/parameters/parameters.json

```diff
@@ -289,6 +289,14 @@
     "recommended_values": "",
     "type": "directorypath"
   },
+  "alert_sounds_path": {
```

Comment:
We could have a more generic path to sounds directory parameter? I could imagine this implementation being used elsewhere. This is more explicit which I imagine the clinical team would like.